### PR TITLE
Update weather cmd desc

### PR DESCRIPTION
**Pull request by robske110**
Requested by @BlackShadow1 and @SalmonGER

### CODE REVIEW

**Comment by MarcoDiGio** [comment]
This can be better if it's "commands.weather.usage=/weather [clear|rain|thunder] < world >"

**Comment by robske110** [comment]
?

**Comment by BlackShadow1** [comment]
Sorry, Github means <world> like as HTML

**Comment by robske110** [comment]
No, it knows its an ini

**Comment by BlackShadow1** [comment]
I don't know nothing of github.

**Comment by BlackShadow1** [comment]
@robske110 i think we need to delete some comment because is off-topic

**Comment by robske110** [comment]
?

**Comment by BlackShadow1** [comment]
@robske110 We mustn't to use this PR as a chat room..

**Comment by SalmonDE** [comment]
@robske110 Why did you add ()? I can't use the /weather command without a specified world.

**Comment by robske110** [comment]
You should be able to use /weather clear as an player

File: src/pocketmine/lang/locale/deu.ini [hunk 1]
```diff
@@ -324,7 +324,7 @@ pocketmine.plugin.deprecatedEvent=Plugin {%0} hat einen Listener für '{%1}' auf
 pocketmine.plugin.eventError="Event '{%0}' konnte nicht zu '{%1}' weitergegeben werden: {%2} in {%3}"
 
 # -------------------- AdditionalLanguage --------------------
-commands.weather.usage=/weather [clear|rain|thunder] 
+commands.weather.usage=/weather [clear|rain|thunder] (<Welt>)
```
Comment (inxomnyaa) [inline src/pocketmine/lang/locale/deu.ini]: ohne die klammern. eigentlich /weather <clear blaaa> [welt]
welt ist optional deshalb

Comment (robske110) [inline src/pocketmine/lang/locale/deu.ini]: No its /weather clear/rain/thunder OPTIONAL::WORLD

Comment (inxomnyaa) [inline src/pocketmine/lang/locale/deu.ini]: Yes. Optional things are in [], necessary arguments are in <>

Und ich meinte ()

**Comment by robske110** [comment]
Now?

**Comment by inxomnyaa** [comment]
Seems good. Merge?

**Comment by robske110** [comment]
Yes

**Pull request merged**